appveyor.yml

```diff
@@ -50,6 +50,14 @@ build_script:
   - ps: >-
       if(($env:APPVEYOR_PULL_REQUEST_HEAD_REPO_NAME -split "/")[0] -eq ($env:APPVEYOR_REPO_NAME -split "/")[0]) {
         Write-warning "Skipping PR build for branch"; Exit-AppveyorBuild
+      } else {
+        npx.cmd yarn@1.15.2 install --frozen-lockfile
+
+        node script/doc-only-change.js --prNumber=$env:APPVEYOR_PULL_REQUEST_NUMBER --prBranch=$env:APPVEYOR_REPO_BRANCH
+        
+        if($DOC_ONLY_CHANGE -eq "true" ) {
```

Comment:
how does `DOC_ONLY_CHANGE` get set here...?

Reply:
doh!  It was there -- I must have deleted it.